iron-collapse.html

```diff
@@ -174,19 +182,23 @@
         // For `auto` we must calculate what is the final size for the animation.
         // After the transition is done, _transitionEnd will set the size back to `auto`.
         if (size === 'auto') {
```

Comment:
Although not well documented, `updateSize` was and still is part of the public API. Thus I vote against changing from `auto` to `''`. Furthermore, `updateSize('auto', true)` on the caller's side adheres more to speaking code principles than `updateSize('', true)`.